{{cookiecutter.__src_folder_name}}/src/flask/flaskapp/pages.py

```diff
@@ -1,6 +1,7 @@
+{% from 'pages_macros.py' import get_all, get_one with context %}
 from flask import Blueprint, redirect, render_template, request, url_for
 
-from . import db, models
+import models
```

Comment:
Doesnt get_all() need the db global for the non-mongo case?

Reply:
yes, added back with postgres context